elasticsearch/src/test/java/org/apache/calcite/adapter/elasticsearch/Projection2Test.java

```diff
@@ -72,7 +78,8 @@ public static void setupInstance() throws Exception {
             "select _MAP['a'] AS \"a\", "
                 + " _MAP['b.a']  AS \"b.a\", "
                 +  " _MAP['b.b'] AS \"b.b\", "
-                +  " _MAP['b.c.a'] AS \"b.c.a\" "
+                +  " _MAP['b.c.a'] AS \"b.c.a\", "
```

Comment:
It would be nice if the users are not forced to specify an alias for every field.

Reply:
Yes I was planning to automatically generate elastic schema based on [Mapping](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping.html)